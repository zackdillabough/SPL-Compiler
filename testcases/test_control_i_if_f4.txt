
a , b ;
{
input a;
input b;
IF b > a {
        a = a+b;
	output a;
	IF a > b { 
		output a;
	}
	a = a+b;
	output a;
}
a = a*b;
output a;
output b;

input a;
input b;

b = b * a;
a = a * b;

output a;
output b;

}
34 12 24 32 9 0 1 3
