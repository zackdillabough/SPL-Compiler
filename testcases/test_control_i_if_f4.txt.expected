408 12 18432 768 